bbc/simorgh
# WSTEAM1-705 link to posts server side (SPIKE)

Resolves JIRA SPIKE [[WSTEAM1-705]](https://jira.dev.bbc.co.uk/browse/WSTEAM1-705)

A simpler version of https://github.com/bbc/simorgh/pull/11242

https://github.com/bbc/simorgh/assets/98817636/eb731123-0dc0-45a9-b328-e38ca9d72a3e



Overall changes
======
Spike to explore Linking to Posts/ Linkable Key Points. Based on PS implementation. See the dropbox paper linked to the ticket.

Code changes
======
- Hard code in some test links
- Passes post param into 'getPageData' function
- Render 'BackToLatest' link above selected post
- Use #post to direct user focus to the 'BackToLatest' link
- Add a Stream Provider
- Modify NextJs app set up so to connect to local BFF

Testing
======

1. Pull down this PR and the correlating BFF PR
2. Connect and run locally
3. Go to http://localhost:7081/pidgin/live/c07zr0zwjnnt
4. See links on the page.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
